docs/k8s-cluster/nvidia-network-operator.md

```diff
@@ -0,0 +1,233 @@
+Deploy Nvidia Network Operator with DeepOps
+===========================================
+
+## Overview
+
+   Nvidia Network Operator leverages Kubernetes CRDs and Operator SDK to manage networking related components in Kuberenets cluster. One of the key components is SR-IOV, which partitions a single PCIe hardware into multiple Virtual Functions (VFs) and attach them directly to Kubernetes pods without going through the virtualization layer on the hosts, thus enables the high performance communication between workloads. High performance networking in Kuberentes also requires a few other components, such as multus-CNI, device drivers and plugins, etc, Nvidia network operator aims to manage all those necessary components automatically under one operator frame work to simply the deployment, operation and management of Nvidia networking for Kubernetes.
+
+Here are the key components that Nvidia network operator try to deploy together:
+
+* SR-IOV Virtual Function (VF) activation
+* Multus CNI
+* SR-IOV CNI for kubernetes
+* SR-IOV device plugin for kubernetes
+* Multus CNI
+* Helm chart for Nvidia network operator
+
+This playbook also install the latest Kubeflow/MPI-Operator for multi-node MPI jobs.
+
+Currently only Infiniband networking is supported in this implementation, RoCE networking support will be added shortly.
+
+
+## Requirements and Tested Environment:
+
+This playbook is developed and tested in following environments:
+
+* Nvidia DGX servers with DGX OS 5.1
+* Mellanox ConnectX-6 VPI HCA
+* Ansible 2.9.27 (deployed by DeepOps)
+* Kubernetes v1.21.6 (deployed by DeepOps)
+* Helm version v3.6.3 (deployed by DeepOps)
+* Nvidia network opertor v1.1.0
+* Infiniband networking. (Ethernet networking support will be added in the future.)
+
+## Deployment Steps
+
+1. Make sure underlying Infiniband network works properly between Kubernetes nodes. It's recommended to run some bare metal micro benchmark testing to verify the IB network is working as expected.
+
+2. Enabling IB port virtualization on IB opensm. This is done in an IB switch in our lab:

```

Comment:
Is this always done on an IB switch? It seems odd to mention "our lab" here, I think we should remove that to conform to DeepOps styleguide.